opba-consent-rest-api/src/main/resources/static/tpp_consent_api.yml

```diff
@@ -1036,3 +1038,27 @@ components:
           format: date
         requestedExecutionTime:
           type: string
+
+    ChallengeData:
```

Comment:
Can we add description to this object, 'Challege data for SCA like OpticTAN image'

Reply:
Added